libshaderc_spvc/src/spvc_private.cc

```diff
@@ -148,7 +169,14 @@ shaderc_spvc_status translate_spirv(shaderc_spvc_context* context,
     opt.RegisterPass(spvtools::CreateGraphicsRobustAccessPass());
```

Comment:
Should this set `registered_pass = true` ?

Reply:
Yup, thanks for the catch.